MAiD - Case Management App/force-app/main/default/objects/Form_1634__c/fields/Potential_BCCS_Review__c.field-meta.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Potential_BCCS_Review__c</fullName>
+    <externalId>false</externalId>
+    <label>Potential BCCS Review</label>
+    <required>false</required>
+    <trackFeedHistory>false</trackFeedHistory>
+    <trackTrending>false</trackTrending>
+    <type>Picklist</type>
+    <valueSet>
+        <restricted>true</restricted>
```

Comment:
@ahshaik1, did you check other fields for Restricted Picklist setting? If others are not active, then this field should not be marked restricted as well.

Reply:
I have checked few other picklist field with values like yes/no or true/false and they are restricted